fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -235,10 +235,29 @@ public static ExecPlan createPhysicalPlan(OptExpression plan, ConnectContext con
                                               List<String> colNames,
                                               TResultSinkType resultSinkType,
                                               boolean hasOutputFragment) {
+        // filter vector function name
+        List<ColumnRefOperator> newOutputColumns = new ArrayList<>();
```

Comment:
Why change the outputColumns here? The rewrite rule should keep the output columns as the old. If you change the output columns, you can add a projec to ensure the new output columns will map to the old one.